hw/xbox/smbus_xbox_smc.c

```diff
@@ -178,9 +179,36 @@ static uint8_t smc_receive_byte(SMBusDevice *dev)
     return 0;
 }
 
+static uint8_t smbus_avpack_to_reg(char *avpack)
+{
+    if (strcmp(avpack, "composite") == 0) {
+        return SMC_REG_AVPACK_COMPOSITE;
+    }
+    if (strcmp(avpack, "scart") == 0) {
+        return SMC_REG_AVPACK_SCART;
+    }
+    if (strcmp(avpack, "svideo") == 0) {
+        return SMC_REG_AVPACK_SVIDEO;
+    }
+    if (strcmp(avpack, "vga") == 0) {
+        return SMC_REG_AVPACK_VGA;
+    }
+    if (strcmp(avpack, "rfu") == 0) {
+        return SMC_REG_AVPACK_RFU;
+    }
+    if (strcmp(avpack, "hdtv") == 0) {
+        return SMC_REG_AVPACK_HDTV;
+    }
+    if (strcmp(avpack, "none") == 0) {
+        return SMC_REG_AVPACK_NONE;
+    }
+    assert(0);
```

Comment:
nit: I think we normally use: `fprintf` to `stderr`, followed by `assert(false)`. But I might misremember or we might have done different things in different files.